mmaction/models/backbones/mobilenetv2.py

```diff
@@ -0,0 +1,199 @@
+    It can be seen here:
+    https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet.py # noqa
+    Args:
+        v (float): original number of channels.
+        divisor (float): Round the number of channels in each layer to
+            be a multiple of this number. Set to 1 to turn off rounding.
+        min_value (int): minimal value to return
+    """
+    if min_value is None:
+        min_value = divisor
+    new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
+    # Make sure that round down does not go down by more than 10%.
+    if new_v < 0.9 * v:
+        new_v += divisor
+    return new_v
+
+
+class InvertedResidual(nn.Module):
+
+    def __init__(self, inp, oup, stride, expand_ratio):
+        """Inverted Residual Mobule from MobilNetV2.
+
+        Args:
+            inp (int): number of input channels.
+            oup (int): number of output channels.
+            stride (int): stride for depthwise convolution.
+            expand_ratio (int): expand ratio for hidden layers.
+        """
+        super(InvertedResidual, self).__init__()
+        self.stride = stride
+        assert stride in [1, 2]
+
+        hidden_dim = int(inp * expand_ratio)
+        self.use_res_connect = self.stride == 1 and inp == oup
+
+        if expand_ratio == 1:
+            self.conv = nn.Sequential(
+                # dw
+                nn.Conv2d(
+                    hidden_dim,
+                    hidden_dim,
+                    3,
+                    stride,
+                    1,
+                    groups=hidden_dim,
+                    bias=False),
+                nn.BatchNorm2d(hidden_dim),
+                nn.ReLU6(inplace=True),
+                # pw-linear
+                nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False),
+                nn.BatchNorm2d(oup),
+            )
+        else:
+            self.conv = nn.Sequential(
+                # pw
+                nn.Conv2d(inp, hidden_dim, 1, 1, 0, bias=False),
+                nn.BatchNorm2d(hidden_dim),
+                nn.ReLU6(inplace=True),
+                # dw
+                nn.Conv2d(
+                    hidden_dim,
+                    hidden_dim,
+                    3,
+                    stride,
+                    1,
+                    groups=hidden_dim,
+                    bias=False),
+                nn.BatchNorm2d(hidden_dim),
+                nn.ReLU6(inplace=True),
+                # pw-linear
+                nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False),
+                nn.BatchNorm2d(oup),
+            )
+
+    def forward(self, x):
+        if self.use_res_connect:
+            return x + self.conv(x)
+        else:
+            return self.conv(x)
+
+
+class MobileNetV2(nn.Module):
+
+    def __init__(self,
+                 width_mult=1.,
+                 inverted_residual_setting=None,
+                 round_nearest=8,
+                 block=None,
+                 pretrained=False):
```

Comment:
Better to make pretrained an url or None, instead of a bool. Since you may use different pretrain models.